Makefile.am

```diff
@@ -178,6 +180,9 @@ anaconda-ci-build:
 anaconda-ci-push:
 	$(CONTAINER_ENGINE) push $(CI_NAME):$(CI_TAG)
 
+anaconda-rpm-build:
+	$(CONTAINER_ENGINE) build $(CONTAINER_BUILD_ARGS) -t $(RPM_NAME):$(CI_TAG) $(RPM_DOCKERFILE)
```

Comment:
I'm missing here possibility to make advanced of the ``ARG`` definition from the Dockerfile.

Reply:
Good point. I'll have to do that anyway in a follow-up to also run rpm-tests for ELN. But for that I'd really like to land #3053 first, so that we have some consistency between the two tests.

Reply:
Actually, no -- this is exactly what `$(CONTAINER_BUILD_ARGS)` is for, and how we use it for the unit-tests ELN run.